roles/bootstrap-os/tasks/bootstrap-oraclelinux.yml

```
@@ -0,0 +1,32 @@
+---
+- name: Download Oracle Linux public yum repo
+  get_url:
+    url: "{{ repo_url }}"
+    dest: "{{ repo_dest }}"
+  vars:
+    repo_dest: /etc/yum.repos.d/public-yum-ol7.repo
+    repo_url: https://yum.oracle.com/public-yum-ol7.repo
+
+- name: Enable Oracle Linux repo
+  ini_file:
+    dest: /etc/yum.repos.d/public-yum-ol7.repo
+    section: "{{ item }}"
+    option: enabled
+    value: 1
+  with_items:
+    - ol7_latest
+    - ol7_addons
+    - ol7_developer_EPEL
+
+- name: Install packages requirements for bootstrap
+  yum:
+    name: "{{ packages }}"
```

Why not directly write the list (or actually: single item) in here?

---

Actually not to a single item, just a single package as there is only one dependency.

---

But you have 2 tasks directly after each other that install one package each...?

---

Yeah, got your point. Changed to one task only. Please check if it looks good now. Thanks.

---

https://docs.ansible.com/ansible/latest/modules/yum_module.html

You can use a list in `name:`, no need to iterate.

```suggestion
 - name: Install packages requirements for bootstrap
   yum:
     name:
       - container-selinux
       - python-pip 
     state: present
```

---

Thanks Markus. Updated to the suggested change. Please check.